src/extension.ts

```diff
@@ -2718,22 +2718,20 @@ function _show_skip_taskbar_windows(ext: Ext) {
             }
 
             let allWindows = global.display.get_tab_list(Meta.TabList.NORMAL_ALL, workspace);
-            let allRunningSkipTaskbarApps = allWindows.filter((w,i,a) => {
-                if (w) {
-                    let found_idx: any;
-                    // Find the first instance using wm_class
-                    for (let index = 0; index < a.length; index++) {
-                        if (a[index].get_wm_class() === w.get_wm_class()) {
-                            found_idx = index;
-                            break;
-                        }
-                    }
-                    return found_idx == i;
-                }
+            // Remove duplicate app names after including skip task bar windows too
+            // E.g. Extensions instance plus when opening an extensions prefs window
+            // Or Android windows when alt-tabbing (depends on where switch apps is bound)
+            let allWindowsWithSkipTaskBar = allWindows.filter((w,i,a) => {
+                let app: any = windowTracker.get_window_app(w);
```

Comment:
Looks good! This is for the Super-Tab app switcher list, I think? Does it also work for Super-Tilde (the key above tab) which switches between windows of the foreground app? Such as if you have a browser with multiple windows. Super-Tilde will scroll through the list of the browser's windows.

Reply:
The `switch-windows for an app` that is usually bound to tilde is not included on the original minimize to tray handling. I do not plan to include it here. `switch-applications` is usually bound to `Super+Tab`. Pop does have Alt+Tab and Super+Tab bound to `switch-applications` by default.

Reply:
@jmmaranan Perfect. I didn't think that Pop Shell was overriding the Super-Tilde but I was just checking if this unintentionally affected the same code (like for example if this "remove duplicate windows" would have removed all extra windows in Super-Tilde, that would have been a bad side effect, hehe). Great to hear that it doesn't affect that code! :)

Reply:
It has to be done for switch windows of this app in the future if more users are using them. But I propose it to be in a separate PR.

Reply:
@jmmaranan Yeah it's too much for this PR. And I personally don't think it matters if Super-Tilde doesn't show minimized windows. It's such a minor thing. How many people even know that super-tilde exists? Not many. :D

Reply:
I agree - when you already have Alt+Tab in muscle memory. Some keyboard layouts like 60-65% have strain (and since I use vim a lot) to get at